index.js

```diff
@@ -150,8 +151,6 @@ alfy.fetch = async (url, options) => {
 		options.responseType = 'json';
 	}
 
-	options.resolveBodyOnly = true;
-
 	let response;
 	try {
 		response = await got(url, options);
```

Comment:
The `transform` function and caching needs to be adjusted when it's `false`.

Reply:
What do you envision those adjustments would look like? A couple thoughts that I have:

- For `transform`, if you're using `resolveBodyOnly: false` then you wouldn't want your return value/data to automatically operate on and return `response.body` – so I think in the case of `resolveBodyOnly: false`, (1) the `transform` fn should operate on the response as is, or (2) or we just disable `transform` application when `resolveBodyOnly: false`. (1) is the current implementation without changing any code.
- For the cache, I have a similar thought. If you want all response data (including `headers`), then I'd think you'd want to cache the whole response as is so that the next time the fetch call is made, you're getting the same response. If you were to, say, store `response.body` in cache then the first time you called `alfy.fetch` with `resolveBodyOnly: false` you'd get a full response object, but the second time you called with the same args you'd get just the body.

Based on these thoughts, I'm not sure what changes to those would look like, other than maybe disabling `transform` when `resolveBodyOnly: false`. Would love to hear your thoughts on that.

Reply:
> For transform, if you're using resolveBodyOnly: false then you wouldn't want your return value/data to automatically operate on and return response.body

I'm fine with it passing in the response, but the behavior should be documented.

> For the cache, I have a similar thought. If you want all response data (including headers), then I'd think you'd want to cache the whole response as is so that the next time the fetch call is made, you're getting the same response. If you were to, say, store response.body in cache then the first time you called alfy.fetch with resolveBodyOnly: false you'd get a full response object, but the second time you called with the same args you'd get just the body.

Agreed.